internal/cli/atlas_alert_config.go

```diff
@@ -18,10 +18,10 @@ import (
 	"github.com/spf13/cobra"
 )
 
-func AtlasAlertConfigBuilder() *cobra.Command {
+func AtlasAlertConfigsBuilder() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "alerts configs",
-		Aliases: []string{"alertConfig", "alertConfigs", "alert-configs", "alert-config"},
+		Use:     "configs",
+		Aliases: []string{"alertConfig", "alertConfigs", "alert-configs", "alert-config", "config"},
```

Comment:
please remove `"alertConfig", "alertConfigs", "alert-configs", "alert-config"` other wise the command would be `alerts alert-config cmd`